games-emulation/fbalpha_libretro/fbalpha_libretro-0.2.97.42.git20180226.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="A port of Final Burn Alpha to the libretro architecture"
+DESCRIPTION="Final Burn Alpha is a multi-systems Arcade Emulator most popular \
+for emulating Neo-Geo, Capcom, Konami, and Cave games. It is developed by the \
+Final Burn team and originated from FinalBurn by Dave."
+HOMEPAGE="https://github.com/libretro/fbalpha"
+COPYRIGHT="2000-2018 the Final Burn team, the libretro team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="c255a8335a8eeee657ae9a0bcb528f14ccc59678"
+SOURCE_URI="https://github.com/libretro/fbalpha/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="b5ff98de2ca9e8e2123d9600ba60f088471c4542b983d79b7a9029d328c0aab3"
+SOURCE_FILENAME="fbalpha-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="fbalpha-$srcGitRev"
+ADDITIONAL_FILES="fbalpha_libretro.info.in"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	fbalpha_libretro$secondaryArchSuffix = $portVersion
+	lib:fbalpha_libretro$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	retroarch$secondaryArchSuffix
```

Comment:
```diff
 REQUIRES="
 	haiku$secondaryArchSuffix
+	"
+SUPPLEMENTS="
 	retroarch$secondaryArchSuffix
 	"
```
(Very few recipes have a [`SUPPLEMENTS`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#supplements) keyword, but this one is a very good example.)

BTW, it builds fine on x86 (both primary and secondary), and x86_64.

Reply:
Interesting feature!

Reply:
Indeed :)

Oops, you forgot to remove the `retroarch$secondaryArchSuffix` from `REQUIRES`.
One of the nice things about `SUPPLEMENTS` is that it allows to create a sort of relation between fbalpha_libretro and retroarch without creating any dependency (on any direction.)

Reply:
Oh so REQUIRES is not necessary when SUPPLEMENTS is present, I see. Lemme fix that.